### ISSUE TITLE
Microphone background audio when switching to other apps

### ISSUE THREAD

**Issue by GitMurf** [Saik0s/Whisperboard#11]
Hi, I was wondering if you planned on implementing ability to have the transcribing / recording still going if you switch to another app? Like in voice memos when you go to another app it shows the orange bar up top indicating something is still recording / using your microphone. Is this something on your radar? Thanks!

**Comment by GitMurf** [comment]
Btw, just noticed the new updates! Looks beautiful! I hadn't tried it in a few weeks :) Was shocked and thought I opened the wrong app 🤣

**Comment by Saik0s** [comment]
Hey @GitMurf, thank you for these nice words. Transcription in the background isn't really doable on iOS. Right now, the app will try to do it for about 30 seconds, but then iOS will suspend the app and decide when to wake it up again to keep going with the transcription. Plus, the memory requirements are pretty high, so the system tries to unload the app as soon as it can.

As for recording in the background, I'll look into that.

**Comment by GitMurf** [comment]
That makes perfect sense on not being able to transcribe in the background. What I was thinking is maybe when app goes to background you pause transcription and instead change to recording the microphone and then when you return to the app, it would transcribe the recorded audio and try to "catch up".

**Comment by seren** [comment]
As a (slightly clunky) workaround, you could use the built-in Voice Memos app to do the recording, save the recording to files (iCloud drive or local), and then import it into WhisperBoard.

**Comment by GitMurf** [comment]
Yeah that is what I currently do. All the other whisper apps like yours have same limitation. It would be a great way for your app to set itself apart :) thanks for quick responses!
